### DISCUSSION TITLE
Solved - Thank you very much! It is a great service you built but I have one question:

### DISCUSSION THREAD

**Discussion by Bui892893knm** [TheLovinator1/discord-twitter-webhooks#17] [General]
Dear Mr, 

I get the following error when the script runs for a couple minutes: 

![image](https://user-images.githubusercontent.com/73953820/119011551-750d3e80-b995-11eb-968d-879b11b1bfc9.png)

All the best

J.B.

**Comment by TheLovinator1** [comment]
Hi Julius, thanks for writing, sorry for the late response.

I fixed (I hope) the connection problem. Tell me if you still get the problem.

I also added USER_LIST_REPLIES_TO_OTHERS_TWEET for replies to other people and USER_LIST_REPLIES_TO_OUR_TWEET for replies to us. 
Works like USERS_TO_FOLLOW:

```
USERS_TO_FOLLOW=12549841489201410,18205090125,852185020125098
USER_LIST_REPLIES_TO_OTHERS_TWEET=12549841489201410,852185020125098
USER_LIST_REPLIES_TO_OUR_TWEET=12549841489201410,852185020125098
```

See [.env.example](https://github.com/TheLovinator1/discord-twitter-webhooks/blob/master/.env.example) for new settings.

Is there anything more you want me to add or change?

~TheLovinator
